control/control.openSUSE.xml

```diff
@@ -418,6 +418,13 @@ textdomain="control"
 
       <system_role>
         <id>custom</id>
+        <software>
+          <!-- "custom" role has an own pattern selection screen mask while
```

Comment:
Again, here I'd write this:

*The "custom" role displays manual the pattern selection dialog in the installation workflow.*